templates/admin/user/view.tmpl

```diff
@@ -0,0 +1,136 @@
+{{template "admin/layout_head" (dict "ctxData" . "pageClass" "admin view user")}}
+
+<div class="admin-setting-content">
+	<div class="admin-responsive-columns">
+		<div class="column">
+			<h4 class="ui top attached header">
+				{{.Title}}
+				<div class="ui right">
+					<a class="ui primary tiny button" href="{{.Link}}/edit">{{ctx.Locale.Tr "admin.users.edit"}}</a>
+				</div>
+			</h4>
+			<div class="ui attached segment">
+				<div class="flex-list">
+					<div class="flex-item">
+						<div class="flex-item-leading">
+							{{ctx.AvatarUtils.Avatar .User 48}}
+						</div>
+						<div class="flex-item-main">
```

Comment:
Is it possiable if you can divide this bug template into multiple `if  possible`.

Reply:
I moved the user details block and the email list out to separate templates. Now, the view template basically only contains the layout, and the content is all being imported from subtemplates.